charts/stable/pydio-cells/values.yaml

```diff
@@ -21,28 +29,57 @@ pydioinstall:
 
 probes:
   liveness:
-    path: "/healthcheck"
-
+    custom: true
+    spec:
+      httpGet:
+        scheme: HTTP
+        path: "/healthcheck"
+        port: 10162
+      failureThreshold: 15
+      initialDelaySeconds: 30
+      periodSeconds: 10
+      timeoutSeconds: 10
   readiness:
-    path: "/healthcheck"
-
+    custom: true
+    spec:
+      httpGet:
+        scheme: HTTP
+        path: "/healthcheck"
+        port: 10162
+      failureThreshold: 15
+      initialDelaySeconds: 30
+      periodSeconds: 10
+      timeoutSeconds: 10
   startup:
-    path: "/healthcheck"
+    custom: true
+    spec:
+      httpGet:
+        scheme: HTTP
+        path: "/healthcheck"
+        port: 10162
+      failureThreshold: 15
+      initialDelaySeconds: 30
+      periodSeconds: 10
+      timeoutSeconds: 10
```

Comment:
this is actually a decrease instead of an increase, please remove these.
The default for the startup probe is about 5 minutes already.